### PR TITLE
NIFIREG-6 Adding nifi-registry-bootstrap module

### PR DESCRIPTION
**Pull request by bbende**
- Enables registry to have the same setup as NiFi where a bootstrap process is started first which launches the main process
- All the standard operations such as start, stop, dump, etc, should now work from nifi-registry.sh
- Added placeholder LICENSE and NOTICE files to the assembly, need to fill in later
- Create nifi-registry-utils to hold utility code

### CODE REVIEW

**Comment by mcgilman** [comment]
Will review...

File: nifi-registry-runtime/src/main/java/org/apache/nifi/registry/NiFiRegistry.java [hunk 1]
```diff
@@ -43,28 +90,242 @@ public void run() {
...
+            LOGGER.info("Jetty web server shutdown completed (nicely or otherwise).");
+        } catch (final Throwable t) {
+            LOGGER.warn("Problem occurred ensuring Jetty web server was properly terminated due to " + t);
+        }
+    }
+
+    /**
+     * Determine if the machine we're running on has timing issues.
+     */
+    private void detectTimingIssues() {
```
Comment (mcgilman) [inline nifi-registry-runtime/src/main/java/org/apache/nifi/registry/NiFiRegistry.java]: I'm not sure we need to detect timing issues for the registry. Can probably remove this check.

**Comment by bbende** [comment]
@mcgilman thanks for reviewing, I made the updates you suggested, let me know if there is anything else.

**Comment by mcgilman** [comment]
Thanks @bbende! This has been merged to master.

**Pull request closed without merging**